jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewTest.java

```diff
@@ -66,4 +91,14 @@ public Void answer( InvocationOnMock invocationOnMock ) throws Throwable {
         verify( currentListGrid ).addColumns( anyList() );
     }
 
+    @Test
+    public void testInitDefaultFilters() {
+        when(filterPagedTable.getMultiGridPreferencesStore()).thenReturn(multiGridPreferencesStore);
+
+        view.initDefaultFilters(new GridGlobalPreferences( "testGrid", new ArrayList<String>(),new ArrayList<String>()), null);
+
+        verify(filterPagedTable,times(3)).addTab((ExtendedPagedTable) any(), anyString(), (Command) any());
+        verify(filterPagedTable,times(3)).saveNewTabSettings(anyString(), (HashMap) any());
+    }
```

Comment:
Instead of casting, you can use any(HashMap.class) or any(ExtendedPageTable.class).